lemur/sources/service.py

```diff
@@ -203,6 +203,10 @@ def sync_certificates(source, user):
                  "gauge", len(certificates),
                  metric_tags={"source": source.label})
 
+    existing_by_id = {}
```

Comment:
this is just a readability suggestion; 
I was wondering, if we can use a more meaning full variable name, since `exists` is also used, but with a different meaning. 

for example: 
`existing_certificates_with_source_by_id`

and: 
```
for cert in certificate_service.get_all_valid_certificates_with_source(source.id)
   existing_certificates_with_source_by_id[cert.id] = cert
```

Reply:
Done.